modules/twinklexfd.js

```diff
@@ -725,7 +725,7 @@ Twinkle.xfd.callbacks = {
 			var usertalkpage = new Morebits.wiki.page('User talk:' + initialContrib, "Notifying initial contributor (" + initialContrib + ")");
 			var notifytext = "\n{{subst:AFDWarning|1=" + Morebits.pageNameNorm + ( params.numbering !== '' ? '|order=&#32;' + params.numbering : '' ) + "}} ~~~~";
 			usertalkpage.setAppendText(notifytext);
-			usertalkpage.setEditSummary("Notification: listing at [[WP:AFD|articles for deletion]] of [[" + Morebits.pageNameNorm + "]]." + Twinkle.getPref('summaryAd'));
+			usertalkpage.setEditSummary("Notification: [[" + params.discussionpage + "|nomination]] of [[:" + Morebits.pageNameNorm + "]]  at [[WP:AFD|articles for deletion]]." + Twinkle.getPref('summaryAd'));
```

Comment:
```suggestion
			usertalkpage.setEditSummary("Notification: [[" + params.discussionpage + "|nomination]] of [[:" + Morebits.pageNameNorm + "]]  at articles for deletion." + Twinkle.getPref('summaryAd'));
```
I think the link to WP:AFD can be skipped (?), to prevent people from accidentally clicking on it and ending up annoyed to find themselves on the AFD main page rather than on the nomination page.

Reply:
I initially thought so too, and had intended on using that space for the discussion link, but I think it's important, especially for newbies and especially for AfD, to provide a link to the actual XfD page.  The AfD notice to the creator doesn't link there at all, and while it may not necessary, I do think at least one link in the edit summary may do more help than harm.